rest-model/src/main/java/com/gentics/mesh/core/rest/node/NodeResponse.java

```diff
@@ -165,8 +167,20 @@ public void setChildrenInfo(Map<String, NodeChildrenInfo> childrenInfo) {
 	 * Return the flag which indicates whether the node is a container.
 	 * 
 	 * @return Container flag
+	 * @deprecated Use {@link #getContainer()}
 	 */
+	@Deprecated
```

Comment:
The other PR removed the method, here it is deprecated. Why?

Reply:
Which PR? #753 is dealing with `BranchResponse#getMigrated`